### PR TITLE
Feature Chill

### PR DESCRIPTION
**Pull request by variableundefined**
## What Does This PR Do

This is a github reminder that we have a feature chill in place.

If you're not a maintainer, to make a major refactor or major feature, you need to ask for a maintainer's approval. For feature / game-changing refactor, get a head onboard with it too.

This is to ensure that we can manage our workload and clear our backlog. People not skilled enough tackling overly big PR occupy a disproportional amount of our time.

## Why It's Good For The Game

Rapid development

## Changelog
:cl:
add: Someone merged in a fake PR. Refer to your nearest OSHA office
/:cl:

### CODE REVIEW

**Comment by Shadow-Quill** [comment]
~~OH GOD OH FUCK~~

**Comment by Shadow-Quill** [comment]
LGTM

File: strings/ion_laws.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "COLD",
         "ICY",
         "SEXY",
```
Comment (Couls) [inline strings/ion_laws.json]: Left sexy in though?

**Comment by Fox-McCloud** [comment]
@Anasari going full Amish now.

**Comment by variableundefined** [comment]
Oops

**Comment by variableundefined** [comment]
It is now a relic of history.

**Pull request closed without merging**